doc/nrf/ug_tfm.rst

```diff
@@ -47,6 +48,18 @@ When building for ``nrf9160dk_nrf9160ns``, UART1 must be disabled in the non-sec
 Otherwise, the non-secure application will fail to run.
 The recommended way to do this is to copy the .overlay file from the :ref:`tfm_hello_world` sample.
 
+.. _tfm_minimal_build:
+
+Minimal build
+=============
+
+The default configuration of TF-M has all supported features enabled, which results in a significant memory footprint.
+A minimal version of the TF-M secure application is provided to show how to configure a reduced version of TF-M.
```

Comment:
@hakonfam  can we add which features are not available in the minimal config as compared with the full version of TF-M?

Reply:
> @hakonfam can we add which features are not available in the minimal config as compared with the full version of TF-M?

"Compared to" is going to be a bit of an overload. I would suggest listing the limited features minimal provides. Pretty much everything is turned off

Reply:
Great, let's approach it from the other side then and list what is available in minimal config.

Reply:
Added `This version supports generating random numbers and reading some areas of secure memory.`